phpunit/class-wp-rest-url-details-controller-test.php

```diff
@@ -505,32 +441,47 @@ public function test_get_title( $html, $expected ) {
 		$this->assertSame( $expected, $actual );
 	}
 
-
-	public function provide_get_title_data() {
+	public function data_get_title() {
 		return array(
-			'no attributes'                  => array(
-				'<title>Testing &lt;title&gt;:</title>',
-				'Testing &lt;title&gt;:',
+
+			// Happy path for default.
+			'default'                        => array(
+				'<title>Testing &lt;title&gt;</title>',
+				'Testing <title>',
 			),
 			'with attributes'                => array(
-				'<title data-test-title-attr-one="test" data-test-title-attr-two="test2">Testing &lt;title&gt;:</title>',
-				'Testing &lt;title&gt;:',
+				'<title data-test-title-attr-one="test" data-test-title-attr-two="test2">Testing &lt;title&gt;</title>',
+				'Testing <title>',
```

Comment:
I definitely think this decoded version (ie: with the tags properly formed) is the response I would expect to see 👍